src/Microsoft.Sbom.Targets/README.md

```diff
@@ -0,0 +1,55 @@
+# SBOM Generation for .NET Projects
+## Microsoft.Sbom.Targets
+This project implements a custom MSBuild task that generates an SBOM using the SBOM API and CLI tool. The MSBuild task binaries along with the associated targets are packaged as a NuGet package and can be consumed within a .NET project. Once installed, an SBOM will automatically be generated upon building the .NET project.
+
+## MSBuild Task Implementation
+The custom MSBuild task is implemented across the following partial classes:
+- `GenerateSbom.cs`
+- `GenerateSbomTask.cs`
+- `SbomCLIToolTask.cs`
+- `SbomInputValidator.cs`
+
+Due to differences in [MSBuild versions](https://learn.microsoft.com/en-us/visualstudio/msbuild/tutorial-custom-task-code-generation?view=vs-2022#create-the-appsettingstronglytyped-project) between Visual Studio and the .Net Core CLI tool, the SBOM generation logic needed to be split into two parts:
+
+1) `GenerateSbomTask.cs` is invoked if the MSBuild version targets the "Core" (.NET Core) runtime bundled with the .NET Core CLI tool. This class utilizes the SBOM API to generate an SBOM.
+
+2) `SbomCLIToolTask.cs` is invoked if the MSBuild version targets the "Full" (.NET Framework) runtime bundled with Visual Studio. Because the SBOM API does not support .NET Framework, this class utilizes the SBOM CLI Tool to generate an SBOM.
+
+Finally, the `Microsoft.Sbom.Targets.targets` file creates a target that will execute the custom MSBuild task. This file will be automatically imported when consuming the NuGet package.
+
+## SBOM Generation Properties
+The custom MSBuild task accepts most of the arguments available for the [SBOM CLI Tool](../../docs/sbom-tool-arguments.md). After the .targets file is imported into a .NET project, the following properties can be set:
+
+| Property | Default Value | Required |
+|-----------------------------------------------------|-------------|---------|
+| `<GenerateSBOM>`                                    | `false`     | No. To enable SBOM generation, set this to true. |
+| `<SbomGenerationBuildDropPath>`                     | `$(OutDir)` | Yes |
```

Comment:
Delete this line and the SbomGenerationManifestDirPath line

Reply:
Done